src/components/common/attachement.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PictureAsPdf from "@material-ui/icons/PictureAsPdf";
+import PhotoIcon from "@material-ui/icons/Photo";
+import DescriptionIcon from "@material-ui/icons/Description";
+import Box from "@material-ui/core/Box";
+
+const Attachment = (props) => {
+    let icon, color;
+
+    switch (props.extension) {
```

Comment:
I think it will be more readable and clear if you'll create a separate component for each type of attachment, and use Material UI's native colors